odoo/addons/base/data/res_currency_data.xml

```diff
@@ -1238,14 +1238,13 @@
             <field name="currency_subunit_label">Chon</field>
         </record>
 
-        <record id="ZWL" model="res.currency">
-            <field name="name">ZWL</field>
-            <field name="full_name">Zimbabwean dollar</field>
-            <field name="symbol">Z$</field>
+        <record id="ZIG" model="res.currency">
```

Comment:
In stable, we should only add a new record.
Then in master, we should only remove the XML id but keep the record if it existed, in an upgrade script